webapp/src/components/sidebar_right/github_items.jsx

```diff
@@ -34,9 +35,24 @@ function GithubItems(props) {
         let number = null;
 
         if (item.number) {
+            let iconType;
+
+            // item is a pull request
+            if (item.pull_request) {
+                iconType = GitPullRequest;
+            } else {
+                iconType = IssueOpened;
+            }
```

Comment:
Nit:
```suggestion
            let iconType = IssueOpened;

            // item is a pull request
            if (item.pull_request) {
                iconType = GitPullRequest;
            }
```

Reply:
What I like about the current style is that it makes the code more readable IMO. It's clear that  it's an either/or and the reader doesn't have to check if `iconType` is defined.

But that is very much a subjective opinion.